CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ If you are interested in contributing, start by reading this document. Please
 also take a look at our [code of conduct](CODE_OF_CONDUCT.md).
 
 If you have any questions at all, do not hesitate to reach out to us on
-[Slack](kanisterio.slack.com).
+[Slack](https://kanisterio.slack.com).
```

Comment:
Can we use the slack invite link instead? https://join.slack.com/t/kanisterio/shared_invite/enQtNzg2MDc4NzA0ODY4LTU1NDU2NDZhYjk3YmE5MWNlZWMwYzk1NjNjOGQ3NjAyMjcxMTIyNTE1YzZlMzgwYmIwNWFkNjU0NGFlMzNjNTk

Reply:
@ihcsim Does that invite expire?

Reply:
+1

Reply:
No, it's the same from the previous README, before it was revamped.

Reply:
@Sagar2366 Please update the URL to the one provided by @ihcsim 👍🏼

Reply:
Updated to share slack invitation link.